src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,39 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+class Block{
+	public static int blockNumber = 123;
+	public Block(int blockNumber){
+		Block.blockNumber = 123;
+	}
+	public  int getNumber(){
+		return blockNumber;
+	}
+}
+class Street{
+	public static String streetName = "abc";
+	public Street(String streetName){
+		Street.streetName = "abc";
+	}
+	public String getName(){
+		return streetName;
+	}
+}
+class Unit{
+	public static int unitNumber = 123;
+	public Unit(int unitNumber) {
+		Unit.unitNumber = 123;
+	}
+	public int getNumber(){
+		return unitNumber;
+	}
+}
+class Postalcode{
+	public static int postalNumber = 123456;
+	public Postalcode(int postalNumber){
+		Postalcode.postalNumber = 654322;
+	}
+	public int getpostal(){
+		return postalNumber;
+	}
+}
```

Comment:
While reviewing this implementation, I got the feeling that you are not very familiar with java. I think it will be best if you can do something with your teammates to brush up your understanding about java as well. 

Keep attempting all the tutorial activities and try harder.